lib/ansible/modules/network/nxos/nxos_vpc_interface.py

```diff
@@ -194,11 +194,14 @@ def get_commands_to_config_vpc_interface(portchannel, delta, config_value, exist
     commands = []
 
     if delta.get('peer-link') is False and existing.get('peer-link') is True:
```

Comment:
Modify this one as well? `if not delta.get('peer-link') and existing.get('peer-link'):`